gsa/src/web/pages/audits/row.js

```diff
@@ -109,14 +109,15 @@ const Row = ({
             {entity.alterable === 1 && (
               <AlterableIcon size="small" title={_('Audit is alterable')} />
             )}
-            {isDefined(scanner) && scanner.scannerType === GMP_SCANNER_TYPE && (
```

Comment:
Oh I think this is wrong in 20.08 already and should include the GREENBONE_SENSOR_TYPE in that version too.

Reply:
Yeah, we forgot to update the audit row when we added the greenbone sensor type